R/get_param.R

```diff
@@ -18,7 +18,7 @@ START.YEAR <- get_param(
 
 pluck_param <- function(x, file, if_null) {
   out <- purrr::pluck(config::get(file = file), !!!x)
-  param <- ui_field(dplyr::last(unlist(x)))
```

Comment:
In addition to removing `ui_field`, there is a little refactoring here. Sorry for the confusion. Let me explain:

The only meaningful change is removing `usethis::ui_field()`. Then I'm removing some intermediate variables that seem complicated -- the code without them seems clearer.